.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
What is this folder? Should this be in this repository?

Reply:
Removed & added to gitignore